website/index.md

```diff
@@ -44,7 +44,7 @@ var Cat = /** @class */ (function () {
 throw Error();
 ```
 
-That's because TypeScript is performing "down-leveling" to convert your modern JavaScript into something that can run on older JavaScript engines. This moves the code around which, without also generating sourcemaps, confuses error stacks and breakpoint locations.
+That's because TypeScript is performing "down-leveling" to convert your modern JavaScript into something that can run on older JavaScript engines. This moves the code around without also generating sourcemaps, which confuses error stacks and breakpoint locations.
```

Comment:
TypeScript _does_ produce sourcemaps. The new sentence implies this is not the case.

```suggestion
That's because TypeScript is performing "down-leveling" to convert your modern JavaScript into something that can run on older JavaScript engines. Because this moves the code around error stacks and breakpoint locations would be incorrect; so sourcemaps are also generated.
```